aptos-move/mvhashmap/src/unit_tests/proptest_types.rs

```diff
@@ -80,6 +85,13 @@ impl<V: Into<Vec<u8>> + Clone> TransactionWrite for Value<V> {
     fn set_bytes(&mut self, bytes: Bytes) {
         self.maybe_bytes = Some(bytes);
     }
+
+    fn as_modification(&self) -> Option<Self>
+    where
+        Self: Sized,
+    {
+        Some(self.clone())
```

Comment:
This has a bit different semantics that the implementation for WriteOp? I think it's better to be consistent and either return None on deletion here or make WriteOp implementation keep deletion op?

Reply:
I haven't played with proptests, what makes something a deletion here? either of maybe_value and maybe_bytes being none?

Reply:
addressed